libs/pmp-web/shared/ui-table/src/lib/components/table/table.component.html

```diff
@@ -1,72 +1,36 @@
-<div *ngIf="tableConfig">
-  <mat-table [dataSource]="tableConfig?.data">
+<ng-container *ngIf="_tableConfig">
```

Comment:
Maybe create a getter

Reply:
what is our position on this subject? Because recently @patryk-zielinski93 said that getters are to slow

Reply:
Ok, I didn't know ;)